content/10. box_2_definitions.md

```diff
@@ -17,6 +17,7 @@ Transfer learning can be supervised (one or both of the source and target domain
 Regularization is an approach to reduce overfitting of models to training data, where a penalty or constraint is added to a model trained on a training dataset to avoid making large prediction errors on the evaluation dataset.
 
 ### Knowledge Graph
+Human knowledge about a domain can be abstracted into a form of network of nodes and edges collectively forming a knowledge graph. In this network any entity of interest (for example a gene, a disease, a protein, or a cell-line) can be represented as a node. All nodes can be linked in a directed manner through edges that represent known relationships between the nodes. For example a gene (node) can be linked to a protein (node) using a directed edge that represents the relationship that the protein is generated through the transcription and translation of the gene. Knowledge graphs serve to integrate data that exist is distributed sources, encode human readable knowledge in machine readable format, and evolve in a flexible manner to integrate new knowledge as it becomes available.
```

Comment:
One sentence per line, and some other suggestions too. Don't love what I wrote about directed edges.

```suggestion
A knowledge graph is a network representation of human knowledge about a domain, abstracted into nodes and edges. 
Any entity of interest (for example a gene, a disease, a protein, or a cell-line) can be represented as a node in a knowledge graph.
All nodes can be linked through edges that represent known relationships between the nodes. 
Edges can be directed, indicating that the order of the nodes is important for encoding the relationship, or undirected.
For example, a gene (node) can be linked to a protein (node) using a directed edge that represents the relationship that the protein is generated through the transcription and translation of the gene. 
Knowledge graphs serve to integrate data that exist is distributed sources, encode human readable knowledge in machine readable format, and evolve in a flexible manner to integrate new knowledge as it becomes available.
```